src/cliverter.ts

```diff
@@ -95,6 +95,6 @@ if (inputArgs.length === 0) {
 			supportedCommands[inputArgs[0]].action(inputArgs[1]);
 		}
 	} else {
-		console.log(`Unrecognized command. See 'cliverter --help'`);
+		console.log(`${inputArgs[0]} ${(inputArgs[1])?inputArgs[1]+" ":""}is not a recognized command. See 'cliverter --help`);
```

Comment:
What if the user runs `cliverter this is a wrong command`? We cannot afford to print `"this is a wrong command" is not a recognized command`

Ideally, we should print: `"this" is not a recognized command`. Because if you pay attention, the first input is the command (conversion type) and the second input is the value.